src/utils.ts

```diff
@@ -96,6 +96,10 @@ export function getTimestampString() {
     return `[${time.getHours()}:${time.getMinutes()}:${time.getSeconds()}]`;
 }
 
+export function sleep(ms: number) {
```

Comment:
Possibly should be `sleepAsync`?

```suggestion
export function sleep(ms: number): Promise<void> {
```

Reply:
Don't know if that pattern has come over from C# but I could add `Async`...

Reply:
Yeah, not sure about the async suffix -- dunno what the current state of play is in the async functions in node's stdlib

Reply:
But the return signature might be a nice addition

Reply:
totally agree!